dropwizard-parent-pom/pom.xml

```diff
@@ -252,11 +251,6 @@
       </dependency>
       <dependency>
         <groupId>io.dropwizard</groupId>
-        <artifactId>dropwizard-spdy</artifactId>
```

Comment:
Our parent pom has more than just dependency versions though -- I'm thinking specifically of the docker configuration that's critical.  Can we merge the two concepts together?

Would this result in this pom looking something like...


```xml
<project>
  ...
  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>io.dropwizard</groupId>
        <artifactId>dropwizard-bom</artifactId>
        <version>${dropwizard.version}</version>
        <type>pom</type>
        <scope>import</scope> <!--  <--- this is the important line -->
      </dependency>
      ...
    </dependencies>
  </dependencyManagement>
  ...
</project>
```